grid-proxy/tools/db/generate.go

```diff
@@ -31,10 +31,10 @@ const (
 	nodeUpRatio          = .5
 	nodeCount            = 1000
```

Comment:
maybe increase the numbers a bit??
let's do nodes 100000, farms 10000, nodeContracts 100000, rentContracts 1000, nameContracts 1000, normalUsers 50000. 
what do you think?